ansible/ci.yml

```diff
@@ -16,5 +16,5 @@
     - galaxy_roles/geerlingguy.java
     - galaxy_roles/geerlingguy.jenkins
     - galaxy_roles/jdauphant.nginx
-    - galaxy_roles/gantsign.maven
-    - galaxy_roles/shelleg.gradle
+    # - galaxy_roles/gantsign.maven
+    # - galaxy_roles/shelleg.gradle
```

Comment:
Using gradle wrapper each project should pull in its own version of gradle so we don't need it globally.